app/repositories/ArrivalMovementRepository.scala

```diff
@@ -106,9 +124,12 @@ class ArrivalMovementRepository @Inject()(mongo: ReactiveMongoApi)(implicit ec:
 
     import models.ArrivalModifier.toJson
 
+    val setLastUpdated         = Json.obj("$set" -> Json.obj("lastUpdated" -> LocalDateTime.now))
```

Comment:
I think we should move this outside in the ArrivalModifier instance

Reply:
@amaalali Ya started doing in ArrivalModifier, there were few concerns around like unit testing and have to place the lastUpdated code in All the different modifiers. Placing it in here helps in future also if we bring some new modifiers we no need to worry about 'lastUpdated'

Reply:
I understand what you're getting at, but this seems to break what ArrivalUpdate and ArrivalModifier is trying to encapsulate, you now need to look in multiple places to see what changes are being applied to a movement